packages/styles/scss/components/card-in-card/_card-in-card.scss

```diff
@@ -44,6 +44,10 @@
     }
 
     .#{$prefix}--card__wrapper {
+      &:hover {
+        transition: all 150ms cubic-bezier(0.2, 0, 0.38, 0.9) 0s;
+      }
```

Comment:
Is this needed? It appears to still work once removed, and you have the transition at the `bx--card` level still. Unless there is something I'm missing here?

```suggestion
```

Reply:
yeah doesn't look like this is necessary anymore